### DISCUSSION TITLE
Is it possible to add "Case created" to the timeline?

### DISCUSSION THREAD

**Discussion by cdennison-dfe** [deejay-hub/timeline-lwc#241] [Q&A] [answered]
I'm new to Timeline so apologies if this question is not appropriate.

I've successfully added tasks, emails, files and custom object, but can't work out a way to display the case created event. Is this possible?

![image](https://user-images.githubusercontent.com/128119310/226257753-abaad031-6c08-430a-9aa6-1fb75a9749be.png)

**Comment by deejay-hub** [comment, marked as answer]
If the Case record is the parent and Time Warp is being added to its page layout then plotting records based on the case Created Date (or any other date) isn't possible. Time Warp can only plot child records to the Parent.

If the parent isn't the case then you can plot related child Case records but it sounds like this isn't the case for you.

**Reply by cdennison-dfe** [reply]
Hi Dave, that makes sense; thanks for getting back to me so quickly. I'll let the customer know.

**Reply by deejay-hub** [reply]
We have an open issue that asks for similar functionality here (https://github.com/deejay-hub/timeline-lwc/issues/174). You can periodically keep an eye on it to see if it gets delivered in a future release. As a free, community driven component we don't make any promises on if/when it might be delivered.
